Yubico.YubiKey/tests/integration/Yubico/YubiKey/Piv/GenerateTests.cs

```diff
@@ -58,7 +58,7 @@ public void GenerateAndSign(PivAlgorithm algorithm)
 
             Assert.True(testDevice.AvailableUsbCapabilities.HasFlag(YubiKeyCapabilities.Piv));
             Assert.True(testDevice is YubiKeyDevice);
-            //             if (testDevice is YubiKeyDevice device) 
+            //             if (testDevice is YubiKeyDevice device)
```

Comment:
remove commented out code

Reply:
Will do this in a future PR when I go through all of the tests @AdamVe 

This code was actually commented out previously